install/kubernetes/Makefile

```diff
@@ -23,22 +35,25 @@ DEV_VERSION_REGEX := '[0-9]\+\.[0-9]\+-dev'
 CILIUM_CHART_REGEX := '\([vV]ersion:\) '$(VERSION_REGEX)
 CILIUM_PULLPOLICY_REGEX := '\([pP]ullPolicy:\) .*'
 QUICK_OPTIONS := \
-    --set operator.replicas=1
+    --set operator.replicas=1 \
+    $(MIN_K8S_OPTIONS)
```

Comment:
nit: maybe move re-arrange a little bit like below, as other flags are more likely to be changed.

```
QUICK_OPTIONS := \
    $(MIN_K8S_OPTIONS) \
    --set operator.replicas=1
    --set operator.replicas=1 \
```